modules/@apostrophecms/form-widget/ui/src/recaptcha.js

```diff
@@ -1,79 +1,50 @@
 /* global grecaptcha */
-let recaptchaSlot;
+let siteKey;
 let recaptchaError;
-let el;
 
 export default function (widgetEl) {
-  if (!widgetEl.querySelector('[data-apos-recaptcha-slot]')) {
+  if (!widgetEl.querySelector('[data-apos-recaptcha-sitekey]')) {
     return;
   }
 
-  recaptchaSlot = widgetEl.querySelector('[data-apos-recaptcha-slot]');
+  siteKey = widgetEl.querySelector('[data-apos-recaptcha-sitekey]').dataset.aposRecaptchaSitekey;
   recaptchaError = widgetEl.querySelector('[data-apos-form-recaptcha-error]');
-  el = widgetEl;
-
-  window.renderCaptchas = render;
 
   if (!document.querySelector('[data-apos-recaptcha-script]')) {
-    addRecaptchaScript();
+
+    window.enableSubmissions = function () {
+      grecaptcha.ready(function() {
+        const buttons = document.querySelectorAll('[data-apos-form-submit]');
+
+        Array.prototype.slice.call(buttons).forEach(btn => {
```

Comment:
Maybe `[... buttons]` works?

Reply:
Yeah it does: https://stackoverflow.com/questions/2735067/how-to-convert-a-dom-node-list-to-an-array-in-javascript